data/ka'het missions.txt

```diff
@@ -0,0 +1,220 @@
+				accept
+
+mission "First Contact: Ka'het: Quarg 2"
+	landing
+	invisible
+	source
+		planet "Alta Hai" "Lagrange" "Kuwaru Efreti"
+	to offer
+		or
+			has "First Contact: Ka'het: Quarg 1A: active"
+			has "First Contact: Ka'het: Quarg 1B: active"
+			has "First Contact: Ka'het: Quarg 1C: active"
+			has "First Contact: Ka'het: Quarg 1D: active"
+	on offer
+		conversation
+			`Do you want to ask the Quarg about the abandoned ringworld in the region of the Graveyard?`
+			choice
+				`	(Yes.)`
+				`	(Not now.)`
+					defer
+			`	Walking down the principal, alien-looking hall, you decide to stop one of the few Quarg you see there. "Hello, human. How may I be of service?" the Quarg says. You reply by asking him about the Graveyard and the ringworld they built there.`
+			`	"Ah, you encountered the remainders of the Builders. Millennia ago, they were a flourish and thoughtful empire, but hostile contacts forced them to grow fearful and hawkish. Although they never did attack us, they always refrained from accepting our aid, and attempted to relocate at a greater distance from us, in worlds easier to maintain than the home they were reserved."`
+			choice
+				`	"Who were the hostile contacts with?"`
+				`	"What happened to the Builders?"`
+					goto ka'het
+			`	"Long ago, few ships coming from a foreign species had reached that region of space, using some of the wormholes leading there. They belonged to a young and foolish race, and they foolishly decided to attack the Builders instead of learning from them. They were stronger any vessel the Builders possessed, but also few enough to be destroyed by the fleets patrolling the area."`
+			choice
+				`	"What happened after that?"`
+			label ka'het
+			`	The Quarg stops for a few seconds before continuing, as if he was reflecting on something. "For many millennia they grew, developing brilliant pieces of technology. Then, compelled by their past experiences, they began creating weapons not only to defend themselves, but also to retaliate; these 'Ka'het', as they named them, defied shortly after they were complete.`
+			`"For few more centuries the Builders lasted, before falling under their very own hubris. Their space is still populated by the Ka'het their gave birth to through the immense facilities they assembled, who do not yet possess a vast intelligence." After saluting you, the Quarg quickly walks away, leaving you with more questions than answers.`
+				decline
+	on decline
+		log "Asked the Quarg about the history of the Graveyard. Although not being told the reason for the abandoned ringworld, have learned about the previous inhabitants of the region, the Builders."
+		log "Factions" "Builders" `The Builders inhabited the Graveyard millennia ago, but have become extinct since. They developed some sort of weapon, called the Ka'het, which stopped obeying their orders and rebelled against them.`
```

Comment:
```suggestion
		log "Factions" "Builders" `The Builders inhabited the Graveyard millennia ago, but have since become extinct. They developed some sort of weapon, called the Ka'het, which stopped obeying their orders and rebelled against them.`
```